### ISSUE TITLE
[WaveCalibrator] SLC FADC clipping level is off by one (Trac #1863)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1855]
The each element of the SLC charge stamp is packed into 9 bits, dropping the LSB if the peak is greater than 512. This makes the charge stamps for high-amplitude SLC hits even by construction, which means that they clip at 1022 instead of 1023. Credit goes to @mzhu and @mrongen for finding this:

https://icecube-spno.slack.com/archives/software/p1475242504000696

WaveCalibrator should mark these bins as clipped so that they can be properly ignored in downstream reconstructions

Migrated from https://code.icecube.wisc.edu/ticket/1863
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:24", 
    "description": "The each element of the SLC charge stamp is packed into 9 bits, dropping the LSB if the peak is greater than 512. This makes the charge stamps for high-amplitude SLC hits even by construction, which means that they clip at 1022 instead of 1023. Credit goes to @mzhu and @mrongen for finding this:\n\nhttps://icecube-spno.slack.com/archives/software/p1475242504000696\n\nWaveCalibrator should mark these bins as clipped so that they can be properly ignored in downstream reconstructions", 
    "reporter": "jvansanten", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067204154158", 
    "component": "combo core", 
    "summary": "[WaveCalibrator] SLC FADC clipping level is off by one", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2016-09-30T14:50:57", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160930T14:52:59: 
* jvansanten commented:

> The compression step is documented in the payload format: https://docushare.icecube.wisc.edu/dsweb/Get/Document-20568


* jvansanten changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20170308T09:27:32: 
* jvansanten commented:

> In changeset 153764:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="153764"
> Marked clipped and undershot regions in InIce SLC launches. Fixes #1855.
> ```


* jvansanten changed status from "accepted" to "closed"
* jvansanten changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:24: 
* olivas commented:

> Milestone October 2016 deleted


* olivas changed milestone from "October 2016" to ""
